10_swirl.Rmd

```diff
@@ -0,0 +1,107 @@
+---
+
+date: "11/21/2021"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Swirl
+
+[Swirl](https://swirlstats.com/) is a handy package for learning R because it is an interactive R package that teaches you how to use R right from the R console!
+
+* Swirl allows you to interactively practice R at your own pace in a variety of lessons
+* Swirl is free, and no login is required!
+
+## Set up Swirl for R practice
+
+### Step 1: Create a new RStudio cloud project
+
+Navigate to [RStudio Cloud](rstudio.cloud). Click on `Your workspace`. We are going to create a separate project to use for practicing R with the Swirl package.
+
+Click on `New Project`. This will take a few seconds to generate.
+
+AVOCADO: needs screenshots!
```

Comment:
````suggestion
```{r, fig.align='center', echo = FALSE, fig.alt= "Put description of image here"}
ottrpal::include_slide("link to google slide here")
```
````